styles/styles.css

```diff
@@ -416,6 +435,93 @@ nav ul li:hover {
   background-size: contain;
 }
 
+.popup {
+  width: 90%;
+  height: 90%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  overflow: hidden;
+  padding: 1rem;
+}
+
+#popup-title {
+  margin-bottom: 40px;
+  position: relative;
+}
+
+.popup-heading {
+  font-family: "Roboto", sans-serif;
+  color: #172b4d;
+  font-size: 32px;
+  line-height: 44px;
+}
+
```

Comment:
- Since you have been working on both versions, mobile and desktop, kindly be sure to make them responsive when you make some changes. Check the project section and align this to the center of the screen on the mobile version

![image](https://user-images.githubusercontent.com/93630700/223745626-3ecf6e67-13d2-47e8-8c61-fabec9d883f8.png)